src/Features/Lsif/GeneratorTest/Utilities/TestLsifOutput.vb

```diff
@@ -87,5 +88,14 @@ Namespace Microsoft.CodeAnalysis.LanguageServerIndexFormat.Generator.UnitTests.U
         Public Async Function GetSelectedRangeAsync() As Task(Of Graph.Range)
             Return (Await GetSelectedRangesAsync()).Single()
         End Function
+
+        Public Function GetFoldingRanges(document As Document) As LSP.FoldingRange()
```

Comment:
Is this only used in FoldingRangeTests? Not sure if it's ideal to avoid littering the core type with something used there, but meh.

Reply:
I added it to match `GetSelectedRangeAsync` helper above.